config/locales/doc_auth/en.yml

```diff
@@ -162,6 +163,10 @@ en:
       secure_account: After you verify, we’ll ask you to encrypt your account.
         Encryption means your data is protected and only you, the account
         holder, will be able to access or change your information.
+      ssn: We need your Social Security number to validate your name, date of birth
+        and address.  Your Social Security number is encrypted.  With your
+        consent, we share this information with the government agency you are
+        trying to access to verify your identity.
```

Comment:
@anniehirshman-gsa I can't tell by the ticket: Should this new text be shown on the initial "Please enter your social security number" screen as well?

Reply:
The implementation ticket to add that text is currently ticketed over in LG-4502.  I agree that we should either have the text on both pages or neither - the "Enter your Social Security number" and "Update your Social Security number" should match except for the heading text and the back button vs. start over/cancel.

@stevegsa would love to review this... would you be able to add screenshots to the PR so I can check out the changes?

Reply:
I posted screenshots.  For some reason I can't put images in the main comment of the pr.